server.toml

```diff
@@ -12,6 +12,7 @@ port = 7788
 tags = [ 'customTag1', 'customTag2', 'customTag3', 'customTag4' ]
 
 resources = [ 
+    'dbg_reconnect',
```

Comment:
Only issue I see here is that `dbg_reconnect` should always be invoked last in the resource list.

This allows for the `core` code pieces to be loaded first, and then reconnect is invoked after those resources have loaded.

Recently the example resources for alt:V I committed a reconnect resource; and that's the case here as well. :)

Reply:
Thank you for your suggestion, I have made a change :D